docs/dashboard_ingestion_guide.md

```diff
@@ -0,0 +1,119 @@
+# Dashboard Ingestion guidance 
+(Currently this guidance is about using Databuilder to ingest Dashboard metadata into Neo4j and Elasticsearch)
```

Comment:
should we put this one under https://github.com/lyft/amundsen/tree/master/docs/tutorials ? Or at least I think we need to make a symlink and make it available there as well.

Reply:
> should we put this one under https://github.com/lyft/amundsen/tree/master/docs/tutorials ? Or at least I think we need to make a symlink and make it available there as well.

I think `symlink` sounds good for it is guidance on using Databuilder, but at the sametime we want it to be in tutorial as well.